src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -52,6 +52,10 @@
 public final class RsWithCookie extends RsWrap {
 
     /**
+     * Matcher expression for non printable chars.
+     */
+    private static final String INVALID_CHARS = "[^\\p{Print}]";
```

Comment:
@popprem not sure this is a good filter. @yegor256 what would be a good validity check here?